Misc/NEWS.d/next/Build/2021-11-25-13-53-36.bpo-45866.ZH1W8N.rst

```diff
@@ -0,0 +1,4 @@
+``make regen-all`` now produces the same output when run from a directory
+other than the source tree: when building Python out of the source tree.
+pegen now strips directory of the "generated by pygen from <FILENAME" header
```

Comment:
```suggestion
pegen now strips directory of the "generated by pygen from <FILENAME>" header
```

Reply:
Oh, that's a backport of a change which already landed in main and 3.10 branches :-( I prefer to keep the typo. The typo can be fixed in a separated PR, but I'm not really motivated to write a PR just for that :-p

Reply:
Well, I created PR to fix this important typo :-) https://github.com/python/cpython/pull/29798/files

cc @hroncok